internal/pkg/cli/env_init.go

```diff
@@ -767,6 +785,33 @@ func (o *initEnvOpts) validateCredentials() error {
 	return nil
 }
 
+func (o *initEnvOpts) validateInternalALBSubnets() error {
+	if o.internalALBSubnets == nil {
+		return nil
+	}
+	isImported := make(map[string]bool)
+	for _, placementSubnet := range o.internalALBSubnets {
+		for _, importedPrivateSubnet := range o.importVPC.PrivateSubnetIDs {
```

Comment:
nit: just syntactic sugar - I think we can reduce the two for-loops into one:
```suggestion
		for _, subnet := range append(o.importVPC.PrivateSubnetIDs, o.importVPC.PublicSubnetIDs...) {
```